src/components/BuildingsTable.vue

```diff
@@ -46,21 +64,105 @@ export default class BuildingsTable extends Vue {
           <th scope="col">Name / Address</th>
           <th scope="col" class="prop-type">Primary Property Type</th>
           <th v-if="showSquareFootage">Square Footage</th>
-          <th v-if="showGasUse" scope="col" class="numeric wide-col">
+          <!-- Click handlers on numeric columns for sorting -->
+          <th
+            v-if="showGasUse"
+            scope="col"
+            class="numeric wide-col"
+            @click="$emit('sort', 'NaturalGasUse')"
+          >
             Fossil Gas Use<br />
             <span class="unit">(kBtu)</span>
+            <button
```

Comment:
I think you need to add an input to the table for `showSort`, since without it, you can see the sort on pages where it doesn't do anything, like [top emitters](https://deploy-preview-174--radiant-cucurucho-d09bae.netlify.app/top-emitters)!

![image](https://github.com/user-attachments/assets/2c17ae93-d854-4be2-8b99-09a61cd120c4)